Traceback (most recent call last):
  File "wgan_gp.py", line 120, in <module>
    os.path.join(os.getcwd(), opt.model_dir, f'discriminator_{opt.maze_type}_{epoch}.pth'))
  File "/home/pierre/miniconda3/envs/teachDRL/lib/python3.6/site-packages/torch/serialization.py", line 376, in save
    with _open_file_like(f, 'wb') as opened_file:
  File "/home/pierre/miniconda3/envs/teachDRL/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/pierre/miniconda3/envs/teachDRL/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/home/pierre/Git/gamegan/src/gan/../../data/models/discriminator_aldous-broder_0.pth'